src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -22,8 +23,8 @@ public Manufacturer create(Manufacturer manufacturer) {
         try (Connection connection = ConnectionUtil.getConnection();
                 PreparedStatement statement
                         = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
-            statement.setString(1, manufacturer.getName());
-            statement.setString(2, manufacturer.getCountry());
+            statement.setObject(1, manufacturer.getName());
+            statement.setObject(2, manufacturer.getCountry());
```

Comment:
probably I missed something, why setObject() is better then setString()?

Reply:
@ArtemPonomar advised me to make it, perhaps this general method provides more flexibility and reduces the risk of errors